docs/core/tools/csproj.md

```diff
@@ -135,7 +135,7 @@ a replacement for the `tools` node in *project.json*.
 ```
 
 #### Version
-`Version` specifies the version of the package to restore. The attribute respect the rules of the NuGet versioning scheme.
+`Version` specifies the version of the package to restore. The attribute respects the rules of the [NuGet versioning](https://docs.microsoft.com/nuget/create-packages/dependency-versions#version-ranges) scheme. The default behavior is an exact version match. For example, specifying `Version="1.2.3"` is equivalent to NuGet notation `[1.2.3]` for the exact 1.2.3 version of the package.
```

Comment:
we're changing the cross-repo links to (/nuget/create-packages/dependency-versions#version-ranges) so they work in offline books.

also, this section is not exactly the one pointed out by @michaelkruglos.